plugins/MauticCrmBundle/Command/PushDataToPipedriveCommand.php

```diff
@@ -77,18 +80,75 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $this->io->text('Pushed '.$pushed);
         }
 
-        $leads = $em->getRepository(Lead::class)->findAll();
         $this->io->title('Pushing Leads');
+
         $leadExport = $this->getContainer()->get('mautic_integration.pipedrive.export.lead');
         $leadExport->setIntegration($integrationObject);
-        $pushed = 0;
-        foreach ($leads as $lead) {
-            if ($leadExport->create($lead)) {
-                ++$pushed;
+
+        $pushed   = 0;
+        $start    = 0;
+        $limit    = 50;
+        $progress = ProgressBarHelper::init($output, $limit);
```

Comment:
The second param should be max count (the count of 100%). 50 is a wild guess of how many integration entities there are. You should count the amount of integration entities that will be processed instead.